validphys2/src/validphys/closuretest/multiclosure_output.py

```diff
@@ -794,3 +853,25 @@ def plot_bias_variance_distributions(
     ax.legend()
     ax.set_title("Total bias and variance distributions.")
     yield fig
+
+@figuregen
+def xq2_data_prcs_maps(xq2_data_map,each_dataset):
```

Comment:
Please add a docstring, I would not know what this function is doing :_

Reply:
I added a description. @giovannidecrescenzo just double check that it describes what the function is supposed to do please